crates/uv/tests/pip_install.rs

```diff
@@ -3484,7 +3482,6 @@ fn respect_no_build_isolation_env_var() -> Result<()> {
 
     ----- stderr -----
     error: Failed to download and build: `anyio @ https://files.pythonhosted.org/packages/db/4d/3970183622f0330d3c23d9b8a5f52e365e50381fd484d08e3285104333d3/anyio-4.3.0.tar.gz`
-      Caused by: Failed to build: `anyio @ https://files.pythonhosted.org/packages/db/4d/3970183622f0330d3c23d9b8a5f52e365e50381fd484d08e3285104333d3/anyio-4.3.0.tar.gz`
       Caused by: Build backend failed to determine metadata through `prepare_metadata_for_build_wheel` with exit status: 1
```

Comment:
This is arguably more ambiguous now, but I think dropping the duplicate package URL is nice. I'm not sure how to improve this in the general case? I think we'd need to implement display for `Error::FetchAndBuild` to include a bit more logic which seems wrong.

Reply:
Don't we have the URL in the line just above? It looks like it was repeated

Reply:
Yeah, it's an improvement not to repeat the URL. The ambiguity here is that it's not clear if the following error, e.g., "Build backend failed to determine metadata", is a _build_ or _download_ error.